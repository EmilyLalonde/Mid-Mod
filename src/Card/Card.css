img {
  height: 200px;
  float: left;
  width: 300px;
}

article {
  border: 1px solid black;
  float: right;
  margin-bottom: 20px;
  margin-right: 5%;
  margin-top: 20px;
  height: 250px;
  width: 90%;
}

h2 {
  margin-top: 30px;
}

p {
  margin-top: 10px;
  margin-bottom: 10px;
}



