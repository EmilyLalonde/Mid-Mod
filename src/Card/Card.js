import React from 'react';
import '../Card/Card.css'

const Card = ({id, img, name, description, price}) => {
  return (
    <article>
      <img src={img} alt='' />
      <h2>{name}</h2>
      <p>{description}</p>
      <h3>${price}</h3>
      <button> Remove From History </button>
    </article>
  )
}

export default Card