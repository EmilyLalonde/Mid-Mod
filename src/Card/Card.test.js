import React from 'react'
import { shallow } from 'enzyme'
import Card from './Card.js'

describe('Card', () => {
  it('renders the name of the purchase in <h2> tags', () => {
    const wrapper = shallow(<Card name="iPhone Xs" />);
    const name = <h2>iPhone Xs</h2>;

    expect(wrapper.contains(name)).toEqual(true);
  });

  it('should match the snapshot', () => {
    const wrapper = shallow(<Card 
      img="https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/iphone-xs-max-gold-select-2018?wid=940&hei=1112&fmt=png-alpha&qlt=80&.v=1550795409154"
      name='iPhone Xs' 
      price = {900}
      description='The latest phone to come out by Apple. It has a faster processor and improved camera to take HDR images.' 
      id={1} 
       />)

    expect(wrapper).toMatchSnapshot();
  });
});