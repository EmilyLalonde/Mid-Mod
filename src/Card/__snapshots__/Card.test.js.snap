// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<article>
  <img
    alt=""
    src="https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/iphone-xs-max-gold-select-2018?wid=940&hei=1112&fmt=png-alpha&qlt=80&.v=1550795409154"
  />
  <h2>
    iPhone Xs
  </h2>
  <h3>
    900
  </h3>
  <p>
    The latest phone to come out by Apple. It has a faster processor and improved camera to take HDR images.
  </p>
  <button>
     Remove From History 
  </button>
</article>
`;
